pkg/server/container_create_windows.go

```diff
@@ -70,11 +70,17 @@ func (c *criService) containerSpec(id string, sandboxID string, sandboxPid uint3
 
 	specOpts = append(specOpts, customopts.WithWindowsResources(config.GetWindows().GetResources()))
 
-	username := config.GetWindows().GetSecurityContext().GetRunAsUsername()
-	if username != "" {
-		specOpts = append(specOpts, oci.WithUser(username))
+	securityContext := config.GetWindows().GetSecurityContext()
```

Comment:
Suggest cache the reference returned from config.GetWindows up on lne 71 and check it for nil...